lib/watch/fork.js

```diff
@@ -20,9 +20,11 @@ process.on('message', function (event) {
   if (event === GRACEFUL_SHUT) {
     const message = chalk.red('[fastify-cli] process forced end')
     setTimeout(exit.bind({ message }), TIMEOUT).unref()
-    fastify.close(() => {
-      process.exit(0)
-    })
+    if (fastify) {
+      fastify.close(() => {
+        process.exit(0)
+      })
+    }
```

Comment:
I think this is missing an `else` clause in which we exit immediately if `fastify` is not there. There is no point in waiting.

Reply:
Good point! In that case, we don't need a `GRACEFUL_SHUT` at all!

Reply:
by adding
`else {
  process.exit(0)
}`
makes it exit without waiting for the message `[fastify-cli] process forced end` to appear.
Is that ok?

Reply:
yes, the app is not started, there is nothing to wait.